stripe/res/values/attrs.xml

```diff
@@ -4,6 +4,7 @@
         <attr name="cardHintText" format="string" />
         <attr name="cardTint" format="color"/>
         <attr name="cardTextErrorColor" format="color"/>
+        <attr name="android:focusedByDefault" />
```

Comment:
what does adding this here do?

Reply:
Basically just tells android that this is an attribute that we care about. Even though this attribute is defined for all views, there doesn't seem to be any way to tell whether it's been explicitly set without adding it to your style. Without this, you can get the value, but you can't tell if it's been set by the user or if android is just returning the default value. Since we want our default value to be different from the system default - i.e. default to true so that we match the prior behaviour - we need to know if the user actually set a value.